src/intents.cs

```diff
@@ -9099,5 +9099,38 @@ interface INVoiceShortcutCenter {
 	//	[Export ("deferredLocalizedIntentsStringWithFormat:fromTable:arguments:", IsVariadic = true)]
 	//	string DeferredLocalizedIntentsStringWithFormat (string format, [NullAllowed] string table, IntPtr arguments);
 	//}
+
+	[Watch (6,0), NoMac, iOS (13,0)]
+	[Native]
+	public enum INCallAudioRoute : long
+	{
+		Unknown = 0,
+		SpeakerphoneAudioRoute,
+		BluetoothAudioRoute,
```

Comment:
Same, just Bluettoh?
```suggestion
		Bluetooth,
```